Cargo.toml

```diff
@@ -17,7 +17,8 @@ serde = { version = "1.0.164", features = ["derive"] }
 serde_json = "1.0.97"
 gix = { version = "0.46.0", features = ["async-network-client", "serde"] }
 gix-ref = { version = "0.30.0", features = ["serde"] }
-tar = { version = "0.4.38", features = ["xattr"] }
+#tar = { version = "0.4.38", features = ["xattr"] }
+tar = { git = "https://github.com/DeterminateSystems/tar-rs.git", branch = "force-mtime", features = ["xattr"] }
```

Comment:
Are we planning to try to upstream this? Otherwise, we'll need to keep this in sync somehow (I don't know how active the upstream is but...)

Reply:
It's pretty ad hoc. Something more general (e.g. a filter to massage the FS metadata in arbitrary ways) might be more acceptable to upstream.

Upstream is not moving super fast: https://github.com/alexcrichton/tar-rs/tags so I don't think it's a lot of effort to rebase from time to time. It's not like the tar format is changing a lot...

Reply:
Let's send a PR anyway and see what they say. We can keep pulling from our fork in the meantime.

Reply:
(That is done in https://github.com/alexcrichton/tar-rs/pull/337)